packages/yourloops/pages/hcp/patients/page.tsx

```diff
@@ -341,7 +160,8 @@ function PatientListPage(): JSX.Element {
 
   if (loading) {
     return (
-      <CircularProgress disableShrink style={{ position: "absolute", top: "calc(50vh - 20px)", left: "calc(50vw - 20px)" }} />
+      <CircularProgress disableShrink
+        style={{ position: "absolute", top: "calc(50vh - 20px)", left: "calc(50vw - 20px)" }} />
```

Comment:
Maybe in another PR ? but it could be a dedicated component without inline style, we should avoid style attribute and use class instead

Reply:
Agreed in another PR. I would change it if it was not already existing code :wink: